new-components/targets/git-clone/testing/integration/git_test.go

```diff
@@ -0,0 +1,191 @@
+package integration
```

Comment:
I'd rather we use a file structure that's more standard for Golang with all the tests in the same package as the component, only split into a separate file with the `// +build !integration` and then invoked with `go test --tags=integration` and all the test related files in the `testdata` folder

Reply:
Fair, I can make this change but let's tackle running the integration tests separately if you don't mind 

Let's leave the same tag for now then we can invest a bit into splitting the execution

Now it runs as a normal test as DIND is on on CI anyway :)